index.js

```diff
@@ -1,97 +1,98 @@
-const book = [];
-
-// saves array of objects into local store
-const saveData = (book) => {
-  localStorage.setItem('savedBooks', JSON.stringify(book));
-};
-
-// Remove object from the array, local store and UI
-const removeBook = (target) => {
-  if (target.className === 'remove') {
-    book.forEach((b) => {
-      if (b.title === target.parentElement.firstElementChild.innerHTML) {
-        const index = book.indexOf(b);
-        book.splice(index, 1);
-        saveData(book);
-        target.parentElement.remove();
-      }
-    });
+// eslint-disable-next-line max-classes-per-file
+class Collection {
+  // eslint-disable-next-line no-unused-vars
+  constructor(books) {
+    this.books = [];
   }
-};
-
-// display book object on UI
-const displayBook = (book) => {
-  if (book) {
-    const display = document.querySelector('.book-display');
-
-    const div = document.createElement('div');
-    div.className = 'book-list';
-
-    display.appendChild(div);
-
-    const title = document.createElement('p');
-    title.textContent = book.title;
-    title.className = 'class-title';
-
-    div.appendChild(title);
-
-    const author = document.createElement('p');
-    author.textContent = book.author;
-    author.className = 'class-title';
-
-    div.appendChild(author);
-
-    // const bookItem = localStorage.setItem('book', JSON.stringify(book));
-
-    const remove = document.createElement('button');
-    remove.textContent = 'Remove';
-    remove.className = 'remove';
 
-    div.appendChild(remove);
-
-    remove.addEventListener('click', (e) => {
-      removeBook(e.target);
-      e.preventDefault();
-    });
+  add(book) {
+    this.books.push(book);
   }
-};
 
-// create book object and add it to array and local store
-const addBook = () => {
-  const bookTitle = document.querySelector('.title');
-  const bookAuthor = document.querySelector('.author');
-  const newBook = {};
-  newBook.title = bookTitle.value;
-  newBook.author = bookAuthor.value;
-
-  // add book to the array
-  book.push(newBook);
+  remove(book) {
+    this.books = this.books.filter((i) => i !== book);
+  }
```

Comment:
It's not removing a book. Removing a book feature isn't working, Kindly fix this 👍🏻